CMakeLists.txt

```diff
@@ -67,15 +67,18 @@ set(nidrivers "nifake" "niscope" "niswitch")
 set(codegen_scripts
   "${codegen_dir}/common_helpers.py"
   "${codegen_dir}/generate_service.py"
-  "${codegen_dir}/handler_helpers.py"
-  "${codegen_dir}/library_interface.h.mako"
-  "${codegen_dir}/library.cpp.mako"
-  "${codegen_dir}/library.h.mako"
-  "${codegen_dir}/mock_library.h.mako"
-  "${codegen_dir}/proto.mako"
+  "${codegen_dir}/service_helpers.py"
```

Comment:
Note: Moved all mako scripts to `templates` folder and included new mako and python helper files here.